config/identity/handler/default.json

```diff
@@ -18,8 +18,16 @@
     {
       "@id": "urn:solid-server:default:IdentityProviderHttpHandler",
       "@type": "IdentityProviderHttpHandler",
+      "idpPath": "/idp",
       "providerFactory": { "@id": "urn:solid-server:default:IdentityProviderFactory" },
-      "interactionHttpHandler": { "@id": "urn:solid-server:auth:password:InteractionHttpHandler" },
+      "templateHandler": {
+        "@type": "TemplateHandler",
+        "templateEngine": { "@type": "EjsTemplateEngine" }
+      },
+      "interactionCompleter": {
```

Comment:
Going to add a comment, but just in case you were wondering: this sends the request back to the OIDC library with the extra metadata we added (specifically: log in in with the given WebID) and then tell it to continue doing its OIDC magic handshake thingy.